nautobot/docs/user-guides/s3-django-storage.md

```diff
@@ -0,0 +1,145 @@
+
+TLDR:
+
+```shell
+$ echo "nautobot[remote_storage]" >> $NAUTOBOT_ROOT/local_requirements.txt
+$ pip3 install "nautobot[remote_storage]"
+```
+
+## Bucket Creation Terraform
+
+The AWS S3 bucket will be hosting Nautobot static files and needs some specific configuration to allow anonymous HTTP access.  The following is an example of Terraform configuration to create the S3 bucket appropriately, the same values can be configured manually:
+
+```terraform
+resource "aws_s3_bucket" "nautobot_static_files" {
+  bucket        = "my-bucket-name"
+  acl           = "public-read"
+
+  cors_rule {
+    allowed_headers = ["*"]
+    allowed_methods = ["GET"]
+    allowed_origins = ["http*"]
+  }
+  cors_rule {
+    allowed_methods = ["GET", "HEAD"]
+    allowed_origins = ["*"]
+  }
+
+  tags = {
+    Name = "Nautobot Static Files"
+  }
+}
+```
+
+## Configuration
+
+The [`django-storages`](https://django-storages.readthedocs.io/en/stable/) library is quite powerful, please refer to their documentation for a more detailed explanation of the individual settings or for more information.
+
+In `nautobot_config.py` define the following configuration:
+
+```python
+STORAGE_BACKEND = "storages.backends.s3boto3.S3Boto3Storage"
+
+STORAGE_CONFIG = {
+    "AWS_ACCESS_KEY_ID": "...",
+    "AWS_SECRET_ACCESS_KEY": "...",
+    "AWS_STORAGE_BUCKET_NAME": "my-bucket-name",
+    "AWS_S3_REGION_NAME": "us-west-1",
+    "AWS_DEFAULT_ACL": "public-read",
+    "AWS_QUERYSTRING_AUTH": False,
+    "AWS_LOCATION": "subfolder/name/static/"
+}
+STATICFILES_STORAGE = STORAGE_BACKEND
+```
+
+If `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are not set, `boto3` [internally looks up IAM credentials](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html).  Using an [IAM Role for EC2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html?icmpid=docs_ec2_console) is highly recommended.
+
+The `STORAGE_CONFIG` has some valuable pieces of information.
+
+- Access Key and Secret Key are for authentication and authorization of the S3 bucket.
+- Storage bucket name is the S3 bucket name, within the bucket provided the aws location will be used to build the parent directory structure.
+    - In the example above the static files will be stored in `f"{AWS_STORAGE_BUCKET_NAME}.s3.amazonaws.com/{AWS_LOCATION}"`
+- Finally, region is where the s3 bucket resides.
+
+!!! info
+    The remaining options are not required, but django-storages documentation should be reviewed to understand the security requirements.
+
+By using the the settings listed above, the static files will be stored in the S3 bucket instead of on the Nautobot servers filesystem. Once the `nautobot-server collectstatic` is executed you will see the files.
```

Comment:
```suggestion
By using the the settings listed above, the static files will be stored in the S3 bucket instead of on the Nautobot server's filesystem. Once the `nautobot-server collectstatic` is executed you will see the files.
```